src/components/TicketCard/TicketCard.js

```diff
@@ -0,0 +1,17 @@
+import { Card } from "@mui/material";
+import "./TicketCard.css";
+
+export default function TicketCard({ content, issuedBy }) {
```

Comment:
file name can be even index.js

Reply:
and index.css 
since you have already mentioned the folder name as TicketCard

Reply:
made suggested changes